foursight_core/app_utils.py

```diff
@@ -248,6 +274,11 @@ def get_domain_and_context(cls, request_dict):
         context = '/api/' if request_dict.get('context', {}).get('path', '').startswith('/api/') else '/'
         return domain, context
 
+    def get_base_path(self, context):
+        # TODO: Have not been able to figure out where/how it is that when running
+        # in production the base path is '/api' and running in locally it is '/'.
+        return "api/" if self.stage.get_stage() != "dev" and "api" not in context else ""
```

Comment:
`dev` or `prod` right?

Reply:
Actually, this should be hooked instead on the (new) is_running_locally function, thanks.